easybuild/easyconfigs/q/QBOX/QBOX-r140b_fix.patch

```diff
@@ -0,0 +1,139 @@
++  //int bhandle = Csys2blacs_handle(comm_);
++  int bhandle = 0;
+   ictxt_ = bhandle;
+   Cblacs_gridinit( &ictxt_, &order, nprow, npcol );
+ 
+-  // get values of nprow_, npcol_, myrow_ and mycol_ in the new context
++  // get values of nprow_
+   if ( ictxt_ >= 0 )
+     Cblacs_gridinfo(ictxt_, &nprow_, &npcol_, &myrow_, &mycol_);
+   
+Only in qball_r140_working/trunk/src: .depend_linux_x86_64_intel
+Only in qball_r140_working/trunk/src: .depend_linux_x86_64_intel.bak
+diff -ru qball_r140/trunk/src/FourierTransform.h qball_r140_working/trunk/src/FourierTransform.h
+--- qball_r140/trunk/src/FourierTransform.h	2013-04-18 20:31:15.000000000 +0200
++++ qball_r140_working/trunk/src/FourierTransform.h	2017-03-10 10:47:52.288415850 +0100
+@@ -34,7 +34,7 @@
+ #include <vector>
+ 
+ #if USE_FFTW || USE_SPIRAL
+-#include "fftw.h"
++#include "fftw3.h"
+ #endif
+ 
+ #include "Timer.h"
+Only in qball_r140_working/trunk/src: .linux_x86_64_gnu.mk.swp
+diff -ru qball_r140/trunk/src/Makefile qball_r140_working/trunk/src/Makefile
+--- qball_r140/trunk/src/Makefile	2013-07-03 22:10:55.000000000 +0200
++++ qball_r140_working/trunk/src/Makefile	2017-03-10 12:40:42.030398018 +0100
+@@ -7,21 +7,53 @@
+ 
+ # If defined, use architecture file set with ARCH variable
+ ARCH ?= $(ARCHGUESS)
+-include $(ARCH).mk
+-#------------------------------------------------------------------------------
+-#
+-TESTEXECS=testMatrix testFTGrid testGridFunction testBasis \
+-testBlacsContext testSlaterDet testEnergyFunctional testSample \
+-testChargeDensity  testFourierTransform testSpecies testContext \
+-testEigenSolvers
+-
+-QBSRC=AtomSet.C Atom.C SymmetrySet.C Symmetry.C Species.C Wavefunction.C SlaterDet.C \
+-      EnergyFunctional.C SampleStepper.C Basis.C FourierTransform.C Matrix.C Context.C \
+-      sinft.C spline.C UnitCell.C StructureFactor.C ChargeDensity.C UserInterface.C RunCmd.C \
+-      LoadCmd.C SaveCmd.C SavesysCmd.C SavedenCmd.C SpeciesCmd.C SpeciesReader.C \
+-      SpeciesHandler.C XCPotential.C LDAFunctional.C PBEFunctional.C PBESolFunctional.C \
+-      PBERevFunctional.C BLYPFunctional.C NonLocalPotential.C SampleReader.C SampleWriter.C \
+-      StructuredDocumentHandler.C SampleHandler.C AtomSetHandler.C WavefunctionHandler.C \
++
++ PLT=LINUX_X86_64_GNU_SALOMON
++#-------------------------------------------------------------------------------
++ #FFTWDIR=/apps/all/FFTW/2.1.5-gompi-2015g
++ FFTWDIR=/apps/all/FFTW/3.3.4-gompi-2015g
++ FFTWLIB=$(FFTWDIR)/lib/libfftw.a
++# BLASDIR=$(HOME)/software/blas/blas-linux_x86-64
++# LAPACKDIR=$(HOME)/software/lapack/lapack-linux_x86-64
++# SCALAPACK_DIR = /apps/all/ScaLAPACK/2.0.2-gompi-2015b-OpenBLAS-0.2.14-LAPACK-3.5.0
++ SCALAPACK_DIR = /apps/all/ScaLAPACK/2.0.2-gompi-2015g-OpenBLAS-0.2.14-LAPACK-3.5.0
++ SCALAPACKLIB  = $(SCALAPACK_DIR)/lib/libscalapack.a
++ #BLASDIR = /apps/all/OpenBLAS/0.2.14-LAPACK-3.5.0/lib
++ BLASDIR = /apps/all/OpenBLAS/0.2.14-GNU-4.9.3-2.25-LAPACK-3.5.0
```

Comment:
this hardcoding isn't going to work in other places, use the corresponding `$EBROOT*` environment variables instead?